polyfills/Element/prototype/childElementCount/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "browsers": {
+    "firefox": "<3.5",
+    "ie": "6 - 8",
+    "opera": "<10.0",
+    "safari": "<4"
+  },
+  "license": "ISC",
+  "repo": "https://github.com/Alhadis/Fix-IE/blob/master/src/IE8-child-elements.js",
```

Comment:
`repo` links should be to the base of the repo, in this case `https://github.com/Alhadis/Fix-IE`.  I appreciate that the link to the file itself may be more useful, but we should follow the schema here